src/backend/distributed/deparser/deparse_comment_stmts.c

```diff
@@ -0,0 +1,78 @@
+/*-------------------------------------------------------------------------
+ *
+ * deparse_coment_stmts.c
+ *
+ *	  All routines to deparse comment statements.
+ *
+ * Copyright (c), Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#include "postgres.h"
+
+#include "catalog/namespace.h"
+#include "commands/defrem.h"
+#include "lib/stringinfo.h"
+#include "nodes/parsenodes.h"
+#include "parser/parse_type.h"
+#include "utils/builtins.h"
+#include "utils/elog.h"
+
+#include "pg_version_compat.h"
+
+#include "distributed/citus_ruleutils.h"
+#include "distributed/commands.h"
+#include "distributed/deparser.h"
+#include "distributed/listutils.h"
+#include "distributed/log_utils.h"
+
+
+typedef struct
+{
+	char *name;
+	int type;
+} ObjectTypeInfo;
+
+const ObjectTypeInfo ObjectTypeNames[] =
```

Comment:
Since this now contains the type of `stmt->object` I think we should rename it to something else. One suggestion is ObjectTypeInfos (I'm not in love with this name but it's better than ObjectTypeNames). Feel free to suggest a better one. 
```suggestion
const ObjectTypeInfo ObjectTypeInfos[] =
```

Reply:
Actually, we can remove the type from this array (see my other comment a little bit below in this file). If we do that, then I think ObjectTypeNames is fine.

Reply:
Changed to ObjectTypeInfos